### ISSUE TITLE
trying to open folder only works once

### ISSUE THREAD

**Issue by UltimateWishub** [photopea/photopea#5725]
trying to open folder only works once

**Comment by photopea** [comment]
Hi, what is opening a folder? What folder?

**Comment by UltimateWishub** [comment]
the feature where you can open from storage
![image](https://github.com/photopea/photopea/assets/60529992/e1f3f599-372b-426d-9950-b8ee67c34b5b)

**Comment by photopea** [comment]
If you gave Photopea access to a folder in your computer, and you want to make it a different folder, press the "X" button at the top right corner of the file browser (the Photopeas one). Is that what you mean?

**Comment by UltimateWishub** [comment]
i dont see the X

**Comment by camoody1** [comment]
> i dont see the X

Hmm... Have you tried looking harder? That usually helps me when I can't find my shoes.

**Comment by photopea** [comment]
What exactly would you like to do? In your screenshot, you are supposed to press "Load This Device" to be able to browser files in your computer inside Photopea.

**Comment by photopea** [comment]
I just wrote an article about storages: https://www.photopea.com/learn/storages

This image has the "X" icon at the top right, which will make Photopa "forget" the storage: https://i.imgur.com/Xp4B3bH.gif

**Comment by UltimateWishub** [comment]
When I try to open folder, the result is the image i sent.

**Comment by photopea** [comment]
Did you click "Load This Device"? Did it ever work for you before? What web browser do you use?

**Comment by UltimateWishub** [comment]
Yes I clicked load this device. It works only once after I clear cookies. Edge stable

**Comment by photopea** [comment]
What you say is so confusing. I wish you just told us what you did. Here is what I did:

1. I just opened www.Photopea.com in MS Edge
2. I clicked "This Device" on the left side
3. I clicked "Load this device" in the middle
4. Edge asked me for a permission, I confirmed, I saw my files in front of me
5. I closed MS Edge
6. I opened MS Edge and opened www.Photopea.com 
7. I clicked "This Device" on the left side
8. I clicked "Load this device" in the middle
9. Edge asked me for a permission, I confirmed, I saw my files in front of me

I can repeat it as many times as I want. Which step exactly is different in your case? What happens instead?

**Comment by UltimateWishub** [comment]
on step 4 this is what happens
![image](https://github.com/photopea/photopea/assets/60529992/870b4c92-0b8a-444a-bc69-ea27f6413038)

**Comment by photopea** [comment]
Could you share a screenshot of the whole screen (after this problem happens), including the browser buttons and everything?

**Comment by UltimateWishub** [comment]
![image](https://github.com/photopea/photopea/assets/60529992/a1553efd-a589-47a9-96ef-8d31b20063c9)

**Comment by photopea** [comment]
Do you know what this icon means? Did you try to click it?
![image (2)](https://github.com/photopea/photopea/assets/15955558/0858611f-19dc-467a-ab02-f94bfb0ba38d)

**Comment by camoody1** [comment]
@UltimateWishub Perhaps recording a short video showing exactly what you're doing and how Photopea is responding would be helpful here. Show what you do from the very start of opening your web browser. From the last post you made, you SHOULD be getting a Windows Explorer popup asking you to select a folder. I use Microsoft Edge, also, and can confirm this works fine for me. Weird suggestion... try NOT being full screen. Maybe the popup is getting lost behind your browser window somehow.🤷🏼‍♂️

**Comment by photopea** [comment]
@UltimateWishub Also, your Photopea seems to be missing ads. I guess you are using many browser extensions.

There are millions of browser extensions, which prevent websites from accessing a local computer, loading files from other domains, etc, to "protect you" from "evil websites".

Please, report only bugs which happen when you run Photopea with no extensions. Otherwise, all your problems could be caused by your browser extensions.

**Comment by UltimateWishub** [comment]
https://github.com/photopea/photopea/assets/60529992/1515a88f-21ef-4626-bbf7-a8e32663926b

**Comment by camoody1** [comment]
> I just did the same what you did in your video (I used MS Edge Incognito on Windows 10). But I was able to see files in my computer.
> 
> Could you please open "about://extensions" in Edge and show us a screenshot? Because you never mentioned how many browser extensions you are using.

His video was done in Private mode. No extensions would have been loaded. There's something weird going on with his PC. Perhaps a virus protection being too strong, or something.

**Comment by photopea** [comment]
You can make extensions work in a private mode in MS Edge, too.

**Comment by UltimateWishub** [comment]
if you see in the video towards the end you will see me click extensions and none were loaded

**Comment by UltimateWishub** [comment]
if it helps, clearing cookies everytime you want to use this feauture is a temp solution.
can you also add option to sort based on date creation.

**Comment by UltimateWishub** [comment]
it seems the built in tracking prevention in edge is what was causing the issue. im not 100% sure but it seems to working well
![image](https://github.com/photopea/photopea/assets/60529992/fdcc0c0f-92c9-42c7-a401-14269a372ea4)
here i turned it off

**Comment by camoody1** [comment]
> it seems the built in tracking prevention in edge is what was causing the issue. im not 100% sure but it seems to working well ![image](https://user-images.githubusercontent.com/60529992/245689523-fdcc0c0f-92c9-42c7-a401-14269a372ea4.png) here i turned it off

Good information to know.

**Comment by photopea** [comment]
Well, if you give Photopea access to your hard drive, it can create an empty file "Photopea.txt" on your hard drive.

Next time you use Photopea, it can see the file "Photopea.txt" and it knows, that you already used Photopea before.

If a website is able to tell, that your device opened that website, or it is opened for the first time, it is called "tracking". Basically, you are tracked, if you let a website store any data in your computer. "Tracking" is not related to "knowing your name", your age or your address, or anything else about the person.

**Comment by photopea** [comment]
So you are tracked, if a website can see some "track" (e.g. a file "Photopea.txt") in your computer, that was created in a previous visit.

But since Edge allows the access just once and never again, a website can create a track once (e.g. a file "Photopea.txt"), but can never see it again to be able to track the user. So it makes total sense.

The wierd part is, that you want to preven tracking, while wanting to let Photopea access your computer more than once :(
